packages/styling/src/styles/DefaultPalette.ts

```diff
@@ -24,30 +326,71 @@ export const DefaultPalette: IPalette = {
   neutralLight: '#eaeaea',
   neutralLighter: '#f4f4f4',
   neutralLighterAlt: '#f8f8f8',
-  accent: '#0078d4',
-  white: '#ffffff',
-  whiteTranslucent40: 'rgba(255,255,255,.4)',
   yellow: '#ffb900',
   yellowLight: '#fff100',
-  orange: '#d83b01',
-  orangeLight: '#ea4300',
   orangeLighter: '#ff8c00',
   redDark: '#a80000',
   red: '#e81123',
   magentaDark: '#5c005c',
   magenta: '#b4009e',
   magentaLight: '#e3008c',
-  purpleDark: '#32145a',
-  purple: '#5c2d91',
   purpleLight: '#b4a0ff',
-  blueDark: '#002050',
   blueMid: '#00188f',
   blue: '#0078d4',
   blueLight: '#00bcf2',
-  tealDark: '#004b50',
   teal: '#008272',
   tealLight: '#00b294',
-  greenDark: '#004b1c',
   green: '#107c10',
   greenLight: '#bad80a',
+
+  // Colors that have no equivalent in Fluent and will be removed.
+  orange: '#d83b01',
+  orangeLight: '#ea4300',
+  purpleDark: '#32145a',
+  purple: '#5c2d91',
+  blueDark: '#002050',
+  tealDark: '#004b50',
+  greenDark: '#004b1c',
+};
+
+// Overrides deprecated colors with the new Fluent colors. This allows for Fluent
```

Comment:
Shouldn't semantic slots be defined in theme.ts under the `_makeSemanticColorsFromPalette` function? Also, consider my changes to deprecation here: https://github.com/OfficeDev/office-ui-fabric-react/pull/4512/files#diff-6118735142300c79db4d510e148e28efR127